glotaran/model/item.py

```diff
@@ -38,6 +38,34 @@
 META_VALIDATOR = "__glotaran_validator__"
 
 
+class GlotaranFieldMetadata(UserDict):
```

Comment:
I know this whole thing isn't really pretty but we need to it keeps us going for now until we reevaluate the need for an extra evaluation mechanism.